content/docs/faqs.md

```diff
@@ -0,0 +1,76 @@
+* **Built-in observability**: Agentgateway comes with built-in metrics and tracing capabilities that allow you to monitor the MCP client and backend tool interactions.
+* **Self-service portal**: Agentgateway provides a built-in self-service developer portal that allows agent developers to easily connect, discover, federate, integrate, and secure agents and tools in any environment, including bare metal, VMs, containers, and Kubernetes.
+* **Open source**: Agentgateway is open source, and licensed under the Apache 2.0 license.
+* **Conformant to the Gateway API project**: Agentgateway is conformant to the Kubernetes Gateway API project, which allows you to use it as a gateway with any Gateway API implementation.
+* **Dynamic configuration updates**: Agentgateway can be updated via an xDS interface without any downtime.
+
+## How does agentgateway relate to kgateway?
+
+While you can manually deploy agentgateway proxies in any environment, you might want a more declarative way to define your agentgateway proxy and deploy it, especially in cloud-native environments, such as Kubernetes. 
+
+The [kgateway open source project](https://kgateway.dev) is the recommended control plane to quickly spin up and manage the lifecycle of agentgateway proxies in Kubernetes clusters. In addition, you can leverage kgateway's traffic management, resiliency, and security policies to further protect your agentgateway proxy and make it more robust. 
+
+For more information about how to deploy agentgateway with kgateway, see the [kgateway documentation](https://kgateway.dev/docs/agentgateway/). 
+
+## What's the difference between agentgateway and kagent? 
+
+Agentgateway provides secure, scalable, stateful, bidirectional connectivity between clients, MCP servers, tools, and agents. However, agentgateway assumes that the MCP servers, tools, and agents already exist in an environment so that it can connect to them. 
+
+To develop, build, and run MCP servers and agents in Kubernetes, you can use the open source project [kagent](https://kagent.dev). Kagent automates complex DevOps and platform engineer operations for you with out-of-the-box agents and tools, intelligent workflows, and built-in troubleshooting.
+
+
+## What license is agentgateway under?
+
+The agentgateway project uses [Apache License 2.0](https://www.apache.org/licenses/).
+
+## What is the project roadmap?
+
+The agentgateway project organizes issues into milestones for each release. For more details, see the following agentgateway links: 
+* [Issues](https://github.com/agentgateway/agentgateway/issues)
+* [Milestones](https://github.com/agentgateway/agentgateway/milestones)
+* [Project board](https://github.com/orgs/agentgateway/projects/1)
+
+## Where is the changelog? 
+
+The changelog is part of each [GitHub release](https://github.com/agentgateway/agentgateway/releases/).
+
+
```

Comment:
Add a section around what if i have more questions, then point users to our community mtg, discord to engage with us.